openapi.yaml

```diff
@@ -739,6 +797,7 @@ components:
             - sek
             - try
         source:
+          unevaluatedProperties: false
```

Comment:
Well, I didn't know this property 🤦

Reply:
It's "new" in JSON Schema 2019-09, which nobody in OAS world had any reason to know about until OAS 3.1, and even then it was not being shouted about because most people dont have a clue what it does. 🤣

https://json-schema.org/understanding-json-schema/reference/object#unevaluatedproperties

Reply:
We don't parse this property for now in Bump, should we still keep it here? Should it have any impact on the generated documentation?

Reply:
@paulRbr no need to do anything with it, its just saying "if you add prperties that arent defined in this bit you'll get a validation error" which is something you should reasonably expect from any API, and you'll know if you screw up because it'll tell you you screwed up.

Reply:
Thanks for the explanation :bow: